lib/main_common.pm

```diff
@@ -991,14 +991,18 @@ sub load_inst_tests {
         {
             # With SLE15 we change desktop using role and not by unselecting packages (Use SYSTEM_ROLE variable),
             # If we have minimalx, as there is no such a role, there we use old approach
-            loadtest "installation/installation_overview_before";
+            loadtest "installation/resolve_dependency_issues";
             loadtest "installation/change_desktop";
         }
     }
     if (get_var("UEFI") && get_var("SECUREBOOT")) {
         loadtest "installation/secure_boot";
     }
     if (installyaststep_is_applicable()) {
+        # Do not repeat step when change_desktop has been executed
```

Comment:
I'm trying to understand this comment. How do you know here that change_desktop has been executed?

Reply:
i dont. the only indication is the new variable. i may remove the comment